frontend/src/store/modules/auth/actions.js

```diff
@@ -130,5 +130,20 @@ export default {
                 reject(err);
             });
         });
-    }
+    },
+    checkEmailUnique: (context, email) => {
+        return new Promise((resolve, reject) => {
+            httpService.post('/auth/unique/email', {
```

Comment:
better to use http.get() this api main purpose to receive status value. not to create, update smth.